src/audio/smart_amp_test.c

```diff
@@ -90,7 +90,7 @@ static int smart_amp_set_config(struct comp_dev *dev,
 	size_t bs;
 
 	/* Copy new config, find size from header */
-	cfg = (struct sof_smart_amp_config *)cdata->data->data;
+	cfg = ASSUME_ALIGNED(cdata->data->data, 4);
```

Comment:
is this 4 words or 4 bytes? Maybe sizeof()?

Also typo in commit title

Reply:
Yes, sizeof(uint32_t) would be more readable. Defining #define member_size(type, member) sizeof(((type *)0)->member) and using as member_size(sof_smart_amp_config, size) could be even better.

Reply:
Fixed.